portfolio/src/main/webapp/index.html

```diff
@@ -180,6 +180,11 @@ <h2>Latest Reports</h2>
         <button onclick="route()">GO SAFE!</button>
     </div>
 
+    <div id="repeatedMarkerUI">
```

Comment:
how do they close this? from the js, it looks like once this is shown, it can't be hidden again